fixtures/pages-nodejs-v2-compat/tests/index.test.ts

```diff
@@ -0,0 +1,54 @@
+import { resolve } from "node:path";
+import { fetch } from "undici";
+import { describe, it } from "vitest";
+import { runWranglerPagesDev } from "../../shared/src/run-wrangler-long-lived";
+
+describe("Pages with Node.js compat v2", () => {
+	describe("with _worker.js file", () => {
+		it("should polyfill `process`", async ({ expect, onTestFinished }) => {
+			const { ip, port, stop } = await runWranglerPagesDev(
+				resolve(__dirname, ".."),
+				"./apps/workerjs-file",
+				["--port=0", "--inspector-port=0"]
+			);
+			onTestFinished(stop);
+			const response = await fetch(`http://${ip}:${port}/`);
+			const body = await response.text();
+			expect(body).toMatchInlineSnapshot(
+				`"_worker.js file, process: _debugEnd,_debugProcess,_eventsCount,_fatalException,_getActiveHandles,_getActiveRequests,_kill,_preload_modules,_rawDebug,_startProfilerIdleNotifier,_stopProfilerIdleNotifier,_tickCallback,abort,addListener,allowedNodeEnvironmentFlags,arch,argv,argv0,assert,availableMemory,binding,chdir,config,constrainedMemory,cpuUsage,cwd,debugPort,dlopen,emit,emitWarning,eventNames,execArgv,execPath,exit,exitCode,features,getActiveResourcesInfo,getMaxListeners,getegid,geteuid,getgid,getgroups,getuid,hasUncaughtExceptionCaptureCallback,hrtime,kill,listenerCount,listeners,loadEnvFile,memoryUsage,off,on,once,pid,platform,ppid,prependListener,prependOnceListener,rawListeners,release,removeAllListeners,removeListener,report,resourceUsage,setMaxListeners,setSourceMapsEnabled,setUncaughtExceptionCaptureCallback,setegid,seteuid,setgid,setgroups,setuid,sourceMapsEnabled,stderr,stdin,stdout,title,umask,uptime,version,versions,env,getBuiltinModule,nextTick"`
```

Comment:
I haven't looked at the implementation but would a sorted array make sense here?

It might be easier to read and wouldn't depend on a (possibly changing?) order

Reply:
Sure - I'll sort it.